src/components/employees/employeeCardGroup.tsx

```diff
@@ -14,14 +14,31 @@ export const EmployeeCardGroup = () => {
   if (!value.employeesSharepoint) return null;
   const sortedData = sortEmployeesData(value.employeesSharepoint);
 
+  let widthBooleans = [true, true, true, true];
```

Comment:
Cant we just use `const widthBooleans = CheckScreen...` 
I do not see the need for the extra [true,...]